src/kirby/components/segment/segmented-chip-control/segmented-chip-control.component.tns.html

```diff
@@ -0,0 +1,7 @@
+<ScrollView orientation="horizontal">
+    <WrapLayout orientation="horizontal">
```

Comment:
Are you sure you want to wrap the items??

Reply:
We have tried removing wrap layout and then output is not as expected. Can we please know reason of not using it?

Reply:
Adjusted as suggested